var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -201,6 +210,15 @@ def setup_run_environment(self, env):
 fi
 
 # run with nrnivmodl in path
-set -x
-'{nrnivmodl}' -incflags '{incflags} '"$2" -loadflags '{loadflags} '"$3" "$1"
+set -xe
+
+
+if [ -n "{nrnivmodlcore_call}" ]; then
+    {nrnivmodlcore_call} "$1"
+    libpath=$(dirname */libcorenrnmech*)
+    extra_loadflags="-L $libpath -l corenrnmech"
```

Comment:
```suggestion
    extra_loadflags="-L $libpath -lcorenrnmech"
```

I was thinking if rpath should be added to the directory containing libcorenrnmech. But I think its already added.